datafusion/physical-expr/src/datetime_expressions.rs

```diff
@@ -783,19 +788,11 @@ mod tests {
         ]);
         assert!(res.is_ok());
```

Comment:
We can add a test case verifying two-argument call equal to three-argument one with epoch.

Reply:
I added this kind of test in `timestamps.slt`

Reply:
We could verify using a query like


```sql
SELECT 
  DATE_BIN('15 minutes', '2022-08-03 14:38:50Z', '1970-01-01T00:00:00Z') = 
  DATE_BIN('15 minutes', '2022-08-03 14:38:50Z')
```

And ensure it is true